src/common/com/intellij/plugins/haxe/ide/projectStructure/ui/HaxeConfigurationEditor.form

```diff
@@ -273,7 +273,7 @@
                           <text resource-bundle="com/intellij/plugins/haxe/HaxeBundle" key="haxe.openfl.arguments"/>
                         </properties>
                       </component>
-                      <component id="39470" class="com.intellij.ui.RawCommandLineEditor">
+                      <component id="39470" class="com.intellij.ui.RawCommandLineEditor" binding="myOpenFLArgumentTextField">
```

Comment:
So it was not connected before? `myOpenFLArgumentTextField` was just useless field? 

BTW, http://stackoverflow.com/questions/27865629/openfl-arguments-cant-be-added